packages/cli-kit/src/bugsnag.ts

```diff
@@ -0,0 +1,52 @@
+import * as path from './path.js'
+import {Interfaces} from '@oclif/core'
+import Bugsnag from '@bugsnag/js'
+import {realpath} from 'fs/promises'
+
+/**
+ * If the given file path comes from within a plugin, return the relative path, plus the plugin name.
+ *
+ * This gives us very consistent paths for errors thrown from plugin code.
+ *
+ */
+export function cleanStackFrameFilePath(
```

Comment:
Nitpick: What about merging this module into [error-handler.ts](https://github.com/Shopify/cli/blob/main/packages/cli-kit/src/node/error-handler.ts) to consolidate or the error handling logic in the same module?